### PR TITLE
Updates to category and recommendedsupplier trackableProperties

### PR DESCRIPTION
**Pull request by mkmbayona**

### CODE REVIEW

File: OrdermentumSDK/Models/RecommendedSupplier.swift [hunk 1]
```diff
@@ -105,9 +105,12 @@ extension RecommendedSupplierDataAttributes: Decodable {
...
 extension RecommendedSupplier: AnalyticsTrackable {
     public var trackableProperties: [String: String]? {
-        var propertiesDictionary: [String: String] = [:]
-        propertiesDictionary["tradingName"] = tradingName
-        propertiesDictionary["supplierId"] = supplierId
-        return propertiesDictionary
+        var properties: [String: String] = [:]
+        properties["supplierName"] = tradingName
+        properties["supplierId"] = supplierId
+        properties["marketplace"] = marketplaceSupplier.toString()
```
Comment (johndagostino) [inline OrdermentumSDK/Models/RecommendedSupplier.swift]: wha?

Comment (mkmbayona) [reply OrdermentumSDK/Models/RecommendedSupplier.swift]: Analytics specs for SUPPLIER_DETAILS_VIEWED have "marketplace" property defined as  "Indicates whether the retailer can order products straight through or not"

That is the property, no?

Comment (bstillitano) [reply OrdermentumSDK/Models/RecommendedSupplier.swift]: make

supplierName into 'name'
supplierId into 'id'

Comment (mkmbayona) [reply OrdermentumSDK/Models/RecommendedSupplier.swift]: @bstillitano Changed to directorySupplierId and directorySupplierName.

File: OrdermentumSDK/Models/Category.swift [hunk 2]
```diff
@@ -66,6 +66,7 @@ extension Category: AnalyticsTrackable {
         var properties: [String: String] = [:]
         properties["categoryId"] = id
         properties["categoryName"] = name
+//        properties["pricesShown"] = true.toString()
```
Comment (bstillitano) [inline OrdermentumSDK/Models/Category.swift]: Remove if unused.

**Pull request merged**